packages/atti-components/src/components/Slider/index.js

```diff
@@ -0,0 +1,104 @@
+import React, {useContext, useEffect, useState} from 'react'
+import PropTypes from 'prop-types'
+import styled, {ThemeContext} from 'styled-components';
+import {map, max, min, minBy, round} from 'lodash';
+import SmallText from "../Typographies/SmallText";
+import SliderRail from "./styles/SliderRail";
+import SliderIndicator from "./styles/SliderIndicator";
+import SliderValue from "./styles/SliderValue";
+import SliderStep from "./styles/SliderStep";
+
+const Slider = ({isStepRestricted, isValueDisplayed, onValueChange, precision, range, steps}) => {
+  const {components: {slider: {sliderRail}}} = useContext(ThemeContext)
+  const [isPressed, setIsPressed] = useState(false)
+  const [indicatorPos, setIndicatorPos] = useState(0)
+  const [xMousePosition, setXMousePosition] = useState(0)
+
+  const pxSliderSize = sliderRail.sizes.width.slice(0, -2)
+  // Add event regarding the indicator motion, remove it when Slider is unmounted
+  useEffect(() => {
+    window.addEventListener('mousemove', moveIndicatorEvent)
```

Comment:
Please check that window exist before accessing it for SSR users

Reply:
Done